examples/extras/chromium_selenium.py

```diff
@@ -1,37 +1,94 @@
 import asyncio
-from chromium import ChromiumLoader  # Import the ChromiumLoader class from chromium.py
+import os
+import json
+from dotenv import load_dotenv
+from chromium import ChromiumLoader  # Import your ChromiumLoader class
```

Comment:
can you import the ScrapegraphAI's chromium loader?

Reply:
I have imported the chromium loader from chromium.py file where it is written its of ScrapegraphAI's itself, I just did the feature addition.

https://github.com/user-attachments/assets/e5a276b3-a48d-4e24-b07f-d938b28adc97

Reply:
but I think that this 
`from chromium import ChromiumLoader  # Import your ChromiumLoader class
`
shoudl be 
`from scrapegraphai.docloaders.chromium import ChromiumLoader  # Import your ChromiumLoader class
`

Reply:
Understood, there was a package confusion that I had, thank you for clearing it out. If there's anything else that needs to be done, I will work on it.